sql-parser/src/main/java/io/crate/sql/ExpressionFormatter.java

```diff
@@ -398,42 +374,36 @@ public String visitCast(Cast node, Void context) {
         @Override
         protected String visitTryCast(TryCast node, Void context) {
             return "TRY_CAST(" + process(node.getExpression(), context) + " AS " + process(node.getType(), context) +
-                   ")";
+                ")";
         }
 
         @Override
         protected String visitSearchedCaseExpression(SearchedCaseExpression node, Void context) {
             ImmutableList.Builder<String> parts = ImmutableList.builder();
             parts.add("CASE");
-            for (WhenClause whenClause : node.getWhenClauses()) {
-                parts.add(process(whenClause, context));
-            }
+            node.getWhenClauses().forEach(when -> parts.add(process(when, context)));
```

Comment:
I think these `forEach` changes go a bit too far. Don't see the benefit of changing this.